tests/Custom/WebDriverTest.php

```diff
@@ -49,15 +51,15 @@ public function testDriverCatchesUpstreamErrorsDuringStart(): void
 
     public function testDriverCatchesUpstreamErrorsDuringStop(): void
     {
-        $driver = $this->createPartialMock(WebdriverClassicDriver::class, ['createWebDriver', 'isStarted', 'getWebDriver']);
-        $driver
-            ->expects($this->once())
-            ->method('isStarted')
-            ->willReturn(true);
-        $driver
-            ->expects($this->once())
-            ->method('getWebDriver')
-            ->willThrowException(new \RuntimeException('An upstream error'));
+        $mockWebDriver = $this->createMock(RemoteWebDriver::class);
+        $mockWebDriverOptions = $this->createMock(WebDriverOptions::class);
+        $mockWebDriverTimeouts = $this->createMock(WebDriverTimeouts::class);
+        $mockWebDriver->method('manage')->willReturn($mockWebDriverOptions);
+        $mockWebDriverOptions->method('timeouts')->willReturn($mockWebDriverTimeouts);
```

Comment:
Please do the same as in `CapabilityTest::testThatCapabilitiesAreAsExpected` test.